### DISCUSSION TITLE
Card Refresh Discussion

### DISCUSSION THREAD

**Discussion by zachiler** [ColorlibHQ/AdminLTE#3518] [General]
I'm not sure exactly how to word this, so I'll do my best. I've been on a journey trying to understand the Card Refresh plugin and how it's supposed to work. Currently, I have a table setup like this in the first div:

`<div class="card" id="card-refresh_widget" data-card-widget="card-refresh" data-source="../tables/somefile.php">`

Then, just for testing, I call the refresh like this:

`<script>
window.onload = function() {
    setInterval(function(){
        $("#card-refresh_widget").CardRefresh("load");
    }, 2000);
}
</script>`

It works, but I'm still not sure I'm doing it right. The documentation doesn't explain it this way and I'm using the latest build of 3.1.0-rc. Other posts were helpful, but I'd still like to know if this is the correct way. Also, I tried to use a 'require' in the PHP file but no matter what path I gave, it said the file didn't exist. Any ideas?

**Comment by danny007in** [comment]
- Check https://adminlte.io/docs/3.1//javascript/card-refresh.html
- Experiment with options
- 😃

**Reply by zachiler** [reply]
That's actually where I started and I couldn't get it to work. The closest I got was an error saying "missing ) after argument list". That's why I wanted to bring up the discussion. It seems others have gotten it to work correctly, I just need a better example to start from.

**Reply by danny007in** [reply]
make this as issue
we will add examples in docs

**Reply by zachiler** [reply]
Thanks. I've added it as an issue. I'll be posting some of my own examples of other plugins in AdminLTE to the Show & Tell board to help others learn how to use them.

**Reply by ajagabos007** [reply]
Please can you help me with your posted examples, I am the same challenge and the documentation on [https://adminlte.io/docs/3.0/javascript/card-refresh.html](https://adminlte.io/docs/3.0/javascript/card-refresh.html) is not helping.
